sjb/generated/test_branch_jenkins_client_plugin.xml

```diff
@@ -165,7 +165,7 @@ for image in &#39;registry.svc.ci.openshift.org/ci/clonerefs:latest&#39; &#39;re
         fi
     done
 done
-clonerefs_args=\${CLONEREFS_ARGS:---repo=openshift,openshift-ansible=master --repo=openshift,aos-cd-jobs=master }
+clonerefs_args=\${CLONEREFS_ARGS:---repo=openshift,openshift-ansible=master --repo=openshift,aos-cd-jobs=master --repo=openshift,origin=release-3.11 --repo=openshift,openshift-ansible=release-3.11 }
```

Comment:
Not sure if this would work - first master is listed for openshift-ansible, then 3.11.

@stevekuznetsov  WDYT?

Reply:
That will error on flag validation

Reply:
after @vrutkovs I took a look and the `clonerefs_args` is getting `---repo=openshift,openshift-ansible=master` by the grandparent of test_branch_jenkins_client_plugin.xml:

```
./sjb/config/common/test_cases/origin_release_with_ecosystem.yml:    - "openshift,openshift-ansible=master"
```

is there a way to remove args added by parents, or do I need to change the parent hierarchy and duplicate the bits from the parent/grandparent minus that particular `sync` for the various jenkins yml ?

Reply:
OK I've grepped through the generate and clonerefs python and believe the answer on being able remove sync's from one's ancestors is "no"

Will update this PR to duplicate what the current parent/grandparent does *MINUS* the sync for openshift-ansible=master

Reply:
check that !!!! ... there are 311 versions of the ancestors ... switching to those